code/modules/organs/organ.dm

```diff
@@ -77,10 +77,10 @@
 	W.damage += damage
 	W.time_inflicted = world.time
 
-/mob/living/carbon/human/var/list/organs = list()
-/mob/living/carbon/human/var/list/datum/organ/external/organs_by_name = list() //Map organ names to organs
-/mob/living/carbon/human/var/list/datum/organ/internal/internal_organs_by_name = list() //So internal organs have less ickiness too
-/mob/living/carbon/human/var/list/grasp_organs = list()
+/mob/living/var/list/organs = list()
```

Comment:
Not sure if this is warranted. Could have unintended consequences.

Reply:
It was done to get slime puddles to fit into the DNA scanner. But yeah it's not ideal. I'll see if I can find another way.